### DISCUSSION TITLE
Best practice for Frigate+ serving multiple locations

### DISCUSSION THREAD

**Discussion by ehn** [blakeblackshear/frigate#18787] [Ask A Question] [answered]
I'm running Frigate across two distinct locations: rural Sweden (temperate, forest, coastal) and Singapore (tropical, urban residential). Given the different environments, climates, flora/fauna and lighting conditions, I'm wondering about the optimal Frigate+ subscription setup.

Would two separate subscriptions provide better model performance for these environments, or would a single subscription work equally well?

I'm happy to support the project with multiple subscriptions if it improves detection accuracy. Just want to make sure I'm setting this up optimally for both sites. The only real downside I see with two separate subscriptions is that I'd have to log in and out when training.

**Comment by hawkeye217** [comment]
While it's somewhat related to your question, Blake has [said this](https://www.reddit.com/r/frigate_nvr/comments/1kwbod9/frigate_multiple_instances/) before about multiple instances:

> As long as you aren't receiving any form of compensation (monetary or otherwise) for operating these instances that is fine. The terms and conditions do NOT permit redistribution or resale of models. You can't include them as a part of a paid service even if you don't charge for them directly.

In terms of training data and how fine tuning from multiple locations would affect your model, I'll let @blakeblackshear answer directly.

**Reply by ehn** [reply]
> While it's somewhat related to your question, Blake has [said this](https://www.reddit.com/r/frigate_nvr/comments/1kwbod9/frigate_multiple_instances/) before about multiple instances:
> 
> > As long as you aren't receiving any form of compensation (monetary or otherwise) for operating these instances that is fine. The terms and conditions do NOT permit redistribution or resale of models. You can't include them as a part of a paid service even if you don't charge for them directly.

Right. These are both my homes. Nothing is being redistributed. :)

I absolutely don't mind getting another subscription. I'm only interested in what will give the best result versus, possibly, the inconvenience of having to switch accounts regularly, depending on which model I'm training. In the past, I have anyway ended up re-upping my subscription because I've run out of credits.

It would be great if you could, within one account, select cameras and train multiple models, though I realize this would add a lot of UI complexity. Then you could also train separate models for different types of cameras. For example, I have a couple of thermal cameras, and perhaps training on snapshots from them isn't great for the model for normal cameras, because objects look quite different.

**Comment by blakeblackshear** [comment, marked as answer]
I think you are probably fine to manage them all within one account. It's still relatively little variation relative to the base model training set which pulls from all users. If it turns out that it makes more sense to separate them out in the future, I can help you move some cameras to a second account.

**Reply by ehn** [reply]
Great! I'll do that for now. Thank you!

**Comment by yousaf465** [comment]
What if I have two different hardware at both of my homes? I am using coral at one location but Nvidia at my second location. how it will work?

**Reply by NickM-27** [reply]
You would need to request separate model trainings, mobiledet for the coral and yolo-nas for Nvidia
